test/test_wordcloud_cli.py

```diff
@@ -24,6 +24,7 @@
 ]
 ARGUMENT_SPEC_REMAINING = [
     ArgOption(cli_name='stopwords', init_name='stopwords', pass_value=temp.name, fail_value=None),
+    ArgOption(cli_name='regexp', init_name='regexp', pass_value=r'\w{2,}', fail_value=r'12('),
```

Comment:
Can you explain the fail_value?

Reply:
Sure. The `fail_value` is an invalid regular expression as the opening bracket is never closed. If you have another value in mind just replace it.

Reply:
oh sorry I misread where this is... never mind, sure, makes sense.